frontend/survey-project/src/app/components/pages/profile-page/ui/general-informations/general-informations.component.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  background-color: #1a1b5e; 
+}
+
+.title {
+  max-width: 800px; 
+  margin: 0 auto 30px auto; 
+  text-align: left; 
+}
+
+.title h1 {
+  font-size: 40px;
+  color: var(--white-color);
+  letter-spacing: 2px;
+  margin: 0;
+}
+
+.general-info-container {
+  background-color: var(--white-color);
+  color: var(--black-color);
+  max-width: 800px; 
+  margin: 0 auto; 
+  padding: 60px 40px; 
+  border-radius: 10px;
+  box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.2);
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+
+  .form-group {
+    margin-bottom: 20px;
+
+    label {
+      display: block;
+      margin-bottom: 8px;
+      font-weight: bold;
+    }
+
+    input {
+      width: 100%;
+      padding: 10px;
+      border: 1px solid #ccc;
+      border-radius: 4px;
+    }
+
+    &.row {
+      display: flex;
+      justify-content: space-between;
+      gap: 1rem;
+      
+      .col {
+        flex: 1;
+        padding: 0.5rem;
+      }
+    }
+  }
+
+  .app-button {
```

Comment:
If you are trying to access <app-button> component, you don't need '.', because it's the html component.
But I think that probably it's working without this style then because this should be targeting anything.